lua/lsp/null-ls/formatters.lua

```diff
@@ -1,29 +1,12 @@
 local M = {}
-local formatters_by_ft = {}
 
 local null_ls = require "null-ls"
 local services = require "lsp.null-ls.services"
 local Log = require "core.log"
 
-local function list_names(formatters, options)
-  options = options or {}
-  local filter = options.filter or "supported"
-
-  return vim.tbl_keys(formatters[filter])
-end
-
 function M.list_supported_names(filetype)
-  if not formatters_by_ft[filetype] then
-    return {}
-  end
-  return list_names(formatters_by_ft[filetype], { filter = "supported" })
-end
-
-function M.list_unsupported_names(filetype)
```

Comment:
I'm not sure if we can, or want to, restore this function, @Tastyep

Reply:
Do keep in mind that I'm currently using the info from null-ls directly that a provider is working, instead of us assuming it works just because we found the executable.

Reply:
It would probably be easier to make a PR on null-ls to expose a method.
I think it was nice being able to see incorrectly configured providers.
If the data is exposed publicly from null-ls we could compare it to the configured providers and get the list from here.